### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Anton-Zhytonbaiev**
[DEMO LINK](https://Anton-Zhytonbaiev.github.io/react_article/)

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -1 +1,28 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title, date, text }) => {
```
Comment (vanst1) [inline src/components/Article.jsx:3]:
```suggestion
export const Article = ({
  title,
  date,
  text,
}) => {
```

**Review by vanst1 — APPROVED** [review]
Well done)

File: src/components/Article.jsx (hunk 1 above)
Comment (alexandra-protyanova) [reply src/components/Article.jsx:3]: agree

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
Let us add prop-types here as well

**Review by alexandra-protyanova — APPROVED** [review]
Well done 🚀

**Pull request closed without merging**